b2share/modules/communities/views.py

```diff
@@ -238,7 +238,7 @@ def __init__(self, **kwargs):
     #     return '', 204
 
     @pass_community
-    #@need_community_permission(read_permission_factory)
+    @need_community_permission(read_permission_factory)
```

Comment:
Double check with @hevp if this is b2share v3 change or part of the v2 updates

Reply:
Commenting out is probably because of DynamicPermission class -HH

Should be commented in our code